src/classes/OPP_OpportunityNaming_NEW_BATCH.cls

```diff
@@ -0,0 +1,100 @@
+/*
+    Copyright (c) 2015, Salesforce.org
```

Comment:
The year should be updated here and below.

Reply:
I took this from the original OPP_OpportunityNaming_NEW_BATCH.cls, which we will replace.  Do you still want it changed?